templates/telemetry_derived/clients_daily_v6/metadata.yaml

```
@@ -0,0 +1,11 @@
+friendly_name: Clients Daily
+description: >
+  A daily aggregate of each Firefox client,
+  partitioned by day
+owners:
+  - relud@mozilla.com
```

this should use my canonical mozilla email, instead of the alias:
```suggestion
  - dthorn@mozilla.com
```